doc/whats_new/v0.24.rst

```diff
@@ -97,6 +97,35 @@ Changelog
   :class:`covariance.GraphicalLassoCV`. `cv_alphas_` and `grid_scores_` will be
```

Comment:
Uhm you have a lot of changes in whats new. Could you revert it and only add your entry

Reply:
ok

Reply:
Thought I had already done that, but that must have slipped through somehow. Now it should be fine.